### PR TITLE
1.0.8 cran revisions

### PR DESCRIPTION
**Pull request by JSchoenbachler**
Based on the notes from Victoria this time:

Please always explain all acronyms in the description text. -> 'RMA'; ...

Please always write package names, software names and API (application
programming interface) names in single quotes in title and description.
e.g: --> 'tximport'
Please note that package names are case sensitive.

Please write references in the description of the DESCRIPTION file in
the form
authors (year) <doi:...>
authors (year) <arXiv:...>
authors (year, ISBN:...)
or if those are not available: authors (year) <https:...>
with no space after 'doi:', 'arXiv:', 'https:' and angle brackets for
auto-linking.
(If you want to add a title as well please put it in quotes: "Title")

You are still setting options(warn=-1) in your function. This is not
allowed. (utils.R line 153)

Please do not install packages in your functions, examples or vignette.
This can make the functions,examples and cran-check very slow.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
-  For RNA-seq data, fetches metadata and raw reads from NCBI SRA,
-  performs standard adapter and quality trimming using Trim Galore (Krueger, <https://github.com/FelixKrueger/TrimGalore>),
-  performs quality control checks using FastQC (Andrews, <https://github.com/s-andrews/FastQC>),
-  quantifies transcript abundances using salmon (Patro et al. 2017, <doi:10.1038/nmeth.4197>) and potentially
-  refgenie (Stolarczyk et al. 2020, <doi:10.1093/gigascience/giz149>),
-  aggregates the results using MultiQC (Ewels et al. 2016, <doi:10.1093/bioinformatics/btw354>),
-  maps transcripts to genes using biomaRt (Durinkck et al. 2009, <doi:10.1038/nprot.2009.97>),
+  'GEOquery' Davis and Meltzer (2007) <doi:10.1093/bioinformatics/btm254>,
+  'ArrayExpress' Kauffmann et al. (2009) <doi:10.1093/bioinformatics/btp354>,
+  'RMA' Irizarry et al. (2003) <doi:10.1093/biostatistics/4.2.249>, and
```
Comment (jakejh) [inline DESCRIPTION]: Robust multi-array average 'RMA' ...

**Comment by jakejh** [comment]
Is anything in the package installing any packages during cran-check?

**Comment by JSchoenbachler** [comment]
Over an initial inspection no, I think we ended up skipping all of those. I'll do some more digging to find out though.

**Comment by JSchoenbachler** [comment]
I just went through and added `skip_on_cran` to a bunch of array tests that could possibly install something or retrieve something from a external library just to be safe.

**Comment by jakejh** [comment]
Please fix RMA in the DESCRIPTION based on my previous comment.

**Comment by JSchoenbachler** [comment]
@jakejh Done!

**Review by jakejh — APPROVED** [review]

**Pull request merged**